CONTRIBUTING.md

```diff
@@ -72,44 +72,12 @@ Testing can be accomplished on systems capable of hosting supported container ru
    to the server under the directory `$HOME/kurl` in your remote server. The install scripts used for testing will be moved
    to your `$HOME` where they should be executed. For further info see the code at [bin/watchrsync.js](https://github.com/replicatedhq/kURL/blob/799db33f66f91b0680facf7c14e1222798021c57/bin/watchrsync.js#L29-L32).
 
-1. Customize your spec by editing `scripts/Manifest`
-
-    To test the `install.sh` script, you will first need to modify the [scripts/Manifest](./scripts/Manifest) file and set the `INSTALLER_YAML` variable to a valid spec.
-    You can use the website https://kurl.sh/ as a tool to help configure your spec.
-    Example:
-    ```bash
-    KURL_URL=
-    DIST_URL=
-    FALLBACK_URL=
-    INSTALLER_ID=
-    REPLICATED_APP_URL=https://replicated.app
-    KURL_UTIL_IMAGE=replicated/kurl-util:alpha
-    KURL_BIN_UTILS_FILE=
-    INSTALLER_YAML="apiVersion: cluster.kurl.sh/v1beta1
-    kind: Installer
-    metadata:
-      name: testing
-    spec:
-      kubernetes:
-        version: 1.25.3
-      weave:
-        version: 2.8.1
-      openebs:
-        version: 3.3.0
-        isLocalPVEnabled: true
-        localPVStorageClassName: default
-      containerd:
-        version: 1.6.9
-      registry:
-        version: 2.8.1"
-    ```
-
-    After modifying the the Manifest, the `make watchrsync` command will automatically build the scripts and upload them to the remote server.
-    You must wait for the message `synced` to test out your changes on the server:
+   You must wait for the message `synced` to test out your changes on the server:
     
-    ![Screenshot 2022-11-06 at 20 06 35](https://user-images.githubusercontent.com/7708031/200198100-19219107-84dd-4631-a0e4-3200ad5feb99.png)
+   ![Screenshot 2022-11-06 at 20 06 35](https://user-images.githubusercontent.com/7708031/200198100-19219107-84dd-4631-a0e4-3200ad5feb99.png)
```

Comment:
I would strongly prefer to keep the example manifest file here.

Is it too verbose?

Reply:
That is not about to be verbose. The problem is not have the example here, the problem is need to make all copy and pastes, run make per make and etc. Keep a sample under a testdata make the life easier. (ihmo)

Regards the explanation it helps described the steps afterwords in more detail because usually first we only want to ensure that we are able to test things out and see the install running and a follow step we will be looking for to know how to test specific things/versions/specs.

Reply:
@laverya I add its purpose and etc in the target.
Also, see that the new target is under the Development section when we run make --help.